jax/_src/distributed.py

```diff
@@ -145,9 +161,20 @@ def initialize(coordinator_address: str | None = None,
   If you are using TPU, Slurm, or Open MPI, all arguments are optional: if omitted, they
   will be chosen automatically.
 
+  The ``spec_detection_method`` may be used to intentionally, automatically select the values for
```

Comment:
nit: The ``spec_detection_method`` may be used to choose a specific method for detecting those distributed arguments.